source/sdk/math.ooc

```diff
@@ -2,6 +2,7 @@ include math
 use math
 
 FLT_EPSILON: extern Float
+DBL_EPSILON: extern Double
```

Comment:
Do we want to have a nice name here, like `DoubleEpsilon: extern (DBL_EPSILON) Double` ? @marcusnaslund ?